helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/AssignmentMetadataStore.java

```diff
@@ -50,33 +52,43 @@
   private Map<String, ResourceAssignment> _globalBaseline;
   private Map<String, ResourceAssignment> _bestPossibleAssignment;
 
+  AssignmentMetadataStore(String zkServers, String clusterName) {
```

Comment:
@jiajunwang Let's rename zkServers to metadataStoreAddrs or something. We do not want close coupling with ZK.